src/entities/user.entity.ts

```diff
@@ -11,12 +11,27 @@ export class User {
   @Column({ unique: true })
   email: string;
 
-  @Column()
+  @Column({ default: 'default' })
   password: string;
 
   @Column({ nullable: true })
   googleId: string;
 
+  @Column({ type: 'varchar', default: 'default' })
+  firstName: string;
+
+  @Column({ type: 'varchar', default: 'default' })
+  lastName: string;
+
+  @Column({ type: 'varchar', default: 'default' })
+  phone: string;
+
+  @Column({ type: 'integer', default: 0 })
+  user: number;
+
+  @ManyToOne(() => ProfileEntity, (profile) => profile.user, { cascade: true })
+  userId: ProfileEntity;
```

Comment:
again why we need user and iserId

Reply:
I need user Id to check equality between profile table and users